helpers/classes.py

```diff
@@ -72,10 +73,16 @@ def get_lists_of_links(self, course):
                 elif r'/jump_to_id/' in link:
                     jump_to_list.add(link.split('/')[2])
                 # All other links starting with http are external links
-                elif link.startswith('http'):
+                elif link.startswith('http') or link.startswith('//'):
+                    self.get_link_with_protocol(link)
                     external_list.add(link)
 
         return (jump_to_list, internal_list, other_course_list, external_list)
+    
+    def get_link_with_protocol(self, link):
+        if link.startswith('//'):
+            link = f"http{'s' if HTTPS else ''}:{link}"
+        return link
```

Comment:
We don't need this - presenting the links (especially external ones) without any protocol modifications is OK.

Reply:
While testing I noticed that the `file://` protocol is getting appended by the browser if no protocol is present. Added this to avoid any confusion.